src/swarm-tui/Swarm/TUI/Model/StateUpdate.hs

```diff
@@ -146,6 +146,15 @@ initPersistentState opts@(AppOpts {..}) = do
   let initRS' = addWarnings initRS (F.toList warnings)
   return (initRS', initUI, initKs)
 
+loadScenarioInfoFromPath ::
```

Comment:
To me the word `load` implies reading something from disk.  Can we name this something like `scenarioInfoFromPath`, `scenarioInfoByPath`, `getScenarioInfoFromPath`, etc?